LondonBritishLibrary/orient/BLorient11601.xml

```diff
@@ -0,0 +1,305 @@
+                                            <persName ref="PRS14317Rowe">W. G. H. Lowe</persName>. 15 Aug. 1936".</desc>
+                                    </item>
+                                </list>
+                            </additions>
+                            <bindingDesc>
+                                <binding contemporary="false" xml:id="binding">
+                                    <decoNote xml:id="b1" type="Boards">European binding.</decoNote>
+                                </binding>
+                            </bindingDesc>
+                        </physDesc>
+                        <history>
+                            <origin>
+                                    <origDate notBefore="1700" notAfter="1799">18th century</origDate>
+                            </origin>
+                            <provenance>According to a note on <locus target="1"/> bought from a person with the name 
+                                <persName ref="PRS14317Rowe">W. G. H. Lowe</persName> on <date when="1936">15 Aug. 1936</date>.</provenance>
+                            <provenance/>
+                        </history>
+                        <additional>
+                            <adminInfo>
+                                <recordHist>
+                                    <source>
+                                        <listBibl type="catalogue">
+                                            <bibl>
+                                                <ptr target="bm:Strelcyn1978BritishLibrary "/>
+                                                <citedRange unit="page">2-3</citedRange>
+                                            </bibl>
+                                        </listBibl>
+                                    </source>
+                                </recordHist>
+                            </adminInfo>
+                        </additional>
+                    <msPart xml:id="p1"><msIdentifier/><msContents><summary/>
+                            <msItem xml:id="p1_i1">
+                                <locus from="1ra" to="45va"/>
+                                <title ref="LIT1315Dodeka"/>
+                                <note>Various explanatory notes in the margins.</note>
+                                <msItem xml:id="p1_i1.1">
+                                    <locus from="1ra" to="7rb"/>
+                                    <title ref="LIT3144Hosea"/>
+                                    <note>The beginning is missing; the text starts with chapter II, 8.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.2">
+                                    <locus from="7rb" to="13vb"/>
+                                    <title ref="LIT3145Amos"/>
+                                    <note>In the upper margin of <locus target="#7r"/>: <foreign xml:lang="gez">አመ፭ለጰግሜን፡ አሞጽ።</foreign></note>
```

Comment:
I think these are extras of the `type="findingAid"`
see https://github.com/BetaMasaheft/Documentation/issues/1804